optimade/server/entry_collections.py

```diff
@@ -13,8 +14,13 @@
 from .mappers import ResourceMapper
 from .query_params import EntryListingQueryParams, SingleEntryQueryParams
 
+try:
+    ci_force_mongo = bool(int(os.environ.get("OPTIMADE_CI_FORCE_MONGO", 0)))
+except Exception:
```

Comment:
Again, I don't think this will ever throw an exception? Or maybe if it can't convert the content to an `int`, at which point it would return a `TypeError`, so maybe change this to:
```suggestion
except TypeError:
```

Reply:
I think it should actually raise a `ValueError`, e.g. 

```>>> int('some_string')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: 'some_string'
```
but I take your point.